pkg/kev/kev.go

```diff
@@ -60,12 +61,36 @@ func InitProjectWithOptions(workingDir string, opts ...Options) error {
 	return nil
 }
 
+func RenderProjectWithOptions(workingDir string, opts ...Options) error {
+	runner := NewRenderRunner(workingDir, opts...)
+	ui := runner.UI
+
+	results, err := runner.Run()
+	if err != nil {
+		printRenderProjectWithOptionsError(ui)
+		return err
+	}
+
+	envs, err := runner.Manifest().GetEnvironments(runner.config.envs)
+	if err != nil {
+		return err
+	}
+
+	printRenderProjectWithOptionsSuccess(ui, results, envs, runner.config.manifestFormat)
+
+	return nil
+}
+
 // Reconcile reconciles changes with docker-compose sources against deployment environments.
 func Reconcile(workingDir string) (*Manifest, error) {
 	m, err := LoadManifest(workingDir)
 	if err != nil {
 		return nil, err
 	}
+
+	// TODO(es) Remove this after render runner is completed
+	m.UI = kmd.NoOpUI()
```

Comment:
Is this meant to be here?

Reply:
I'll update the TODO, but for now this is required while legacy `reconcile` cmd is still used in the `dev` cmd.